Userland/Utilities/disasm.cpp

```diff
@@ -16,7 +19,24 @@
 #include <LibMain/Main.h>
 #include <LibX86/Disassembler.h>
 #include <LibX86/ELFSymbolProvider.h>
-#include <string.h>
+
+struct Symbol {
+    size_t value;
+    size_t size;
+    StringView name;
+
+    size_t address() const { return value; }
+    size_t address_end() const { return value + size; }
```

Comment:
Same here. FlatPtr for these raw addresses? (Or VirtualAddress if we even have that in userspace :thonk: ?)